publish_data.go

```diff
@@ -61,34 +61,59 @@ func (dp *DataPublisher) RemoveLJH22() {
 	dp.LJH22.Close()
 	dp.LJH22 = nil
 }
+
 func (dp *DataPublisher) HasPubRecords() bool {
 	return dp.PubRecords != nil
 }
 
+// SetPubRecords starts publishing records with czmq over tcp at port=PortTrigs
```

Comment:
Yeah, you really should start clearing out all the Lint warnings about lack of doc strings. It's not like it will be easier later, once you've forgotten what all the functions do.